src/main/resources/templates/admin.html

```diff
@@ -12,61 +12,65 @@
-                            <span>longitude1</span>
-                            <span>Whales1</span>
-                            <span>User1</span>
-                        </div>
+                <div th:if="${sightings!=null and !sightings.empty}">
 
-                        <div class="checkboxgroup">
-                            <label for="radio-button-id2">Reject</label>
-                            <input type="radio" name="radio1" value="reject" id="radio-button-id2" />
-                        </div>
-                        <div class="checkboxgroup">
-                            <label for="radio-button-id1">Approve</label>
-                            <input type="radio" name="radio1" value="approve" id="radio-button-id1" />
-                        </div>
+                    <form th:action="@{/admin}" method="post" th:object="${sightings}">
 
-                    </div>
+                        <div class="sightings-pending-admin-form" th:each="sighting: ${sightings}">
 
-                    <div class="sightings-pending-admin-form">
+                            <div class="sighting">
+                                <div class="checkboxgroup">
+                                    <span>Latitude</span>
+                                    <span th:text="${sighting.getLatitude()}"></span>
+                                </div>
+                                <div class="checkboxgroup">
+                                    <span>Longitude</span>
+                                    <span th:text="${sighting. getLongitude()}"></span>
+                                </div>
 
-                        <div class="sighting">
-                            <span>latitute2</span>
-                            <span>longitude2</span>
-                            <span>Whales2</span>
-                            <span>User2</span>
-                        </div>
+                                <div class="checkboxgroup">
+                                    <span>Whale Type </span>
+                                    <span>Whales1</span>
+                                </div>
+                                <div class="checkboxgroup">
+                                    <span>User ID </span>
+                                    <span th:text="${sighting.getId()}"></span>
+                                </div>
+                            </div>
+
+                            <div class="checkboxgroup">
+                                <label for="radio-button-id1">Reject</label>
+                                <input type="radio" th:name="${sighting.getId()}" value="reject" id="radio-button-id1" />
+                            </div>
+                            <div class="checkboxgroup">
+                                <label for="radio-button-id2">Approve</label>
+                                <input type="radio" th:name="${sighting.getId()}" value="approve" id="radio-button-id2"} />
```

Comment:
I think you don't need/want the } at the end of this line

Reply:
indeed :)